apidoc/Titanium/UI/View.yml

```diff
@@ -404,9 +404,15 @@ events:
       - name: force
         summary: |
             The current force value of the touch event.
-            Note: This property is only available for iOS devices that support 3D-Touch and run 9.0 or later.
+            Note: This property is only available for iOS devices that support 3D-Touch and run 9.0 or later and on some android devices
         type: Number
-
+        
+      - name: size
+        summary: |
+            The current size of the touch area.
+            Note: This property is only available android devices
```

Comment:
The rule to follow there is, that the text per line doesn't extend a certain length (I mostly just compare it to existing lines). It doesn't change much, just a thing that I'm personally used to